qutebrowser/browser/webkit/webkitelem.py

```diff
@@ -128,13 +128,16 @@ def set_value(self, value):
             value = javascript.string_escape(value)
             self._elem.evaluateJavaScript("this.value='{}'".format(value))
 
-    def dispatch_event(self, event):
+    def dispatch_event(self, event, bubbles=False,
+                       cancelable=False, composed=False):
         self._check_vanished()
-        if self._tab.is_deleted():
-            raise webelem.OrphanedError("Tab containing element vanished")
         log.webelem.debug("Firing event on {!r} via javascript.".format(self))
-        self._elem.evaluateJavaScript("this.dispatchEvent(new Event('{}'))"
-                                      .format(event))
+        event = javascript.string_escape(event)
+        self._elem.evaluateJavaScript(
+            "this.dispatchEvent(new Event('{}', "
+            "{{'bubbles': {}, 'cancelable': {}, 'composed': {}}}))"
+            .format(event, str(bubbles).lower(), str(cancelable).lower(),
+                    str(composed).lower()))
```

Comment:
You should probably use `javascript.convert_js_arg()` for all args (including `event`) here instead (feel free to make it public)